nixos/tests/all-tests.nix

```diff
@@ -67,6 +67,7 @@ in
   deluge = handleTest ./deluge.nix {};
   dhparams = handleTest ./dhparams.nix {};
   dnscrypt-proxy2 = handleTestOn ["x86_64-linux"] ./dnscrypt-proxy2.nix {};
+  doas = handleTestOn ["x86_64-linux"] ./doas.nix {};
```

Comment:
Is there any reason for the test to be x86-64 only? The sudo one isn't.

Reply:
Without this, would it attempt to run tests on `darwin` if available? I was under that impression, but if it's unnecessary, I can definitely remove it. A brief search didn't lead me to any documentation on when I should use `handleTestOn` vs `handleTest`.

Reply:
So, I changed it to `handleTest` locally, but `nix-build ./nixos/release.nix -A tests.doas` complained:

```
error: a 'aarch64-linux' with features {} is required to build '/nix/store/drg04kmwswbv0p87wih9i3h5xis4v9s7-append-initrd-secrets.drv', but I am a 'x86_64-linux' with features {benchmark, big-parallel, kvm, nixos-test}
```

Reply:
Sadly NixOS doesn't run on Darwin yet :p

I think that error might be normal if you don't have an AArch64 builder, though I'm not certain. I do `nix build -f . nixosTests.<name>` to run tests locally.

Reply:
Oh. Duh. Absolutely right. `handleTest` it is!

`nix-build -A nixosTests.doas` works fine, thanks for the tip!